connector/pkg/JSONLoader.go

```diff
@@ -50,8 +49,14 @@ func saveJson(data []byte, path string) {
 	defer jsonFile.Close()
 
 	var prettyJSON bytes.Buffer
+
 	err = json.Indent(&prettyJSON, data, "", "    ")
+	if err != nil {
+		log.Panic(err)
```

Comment:
It's better to return error in function and delegates its handling to a higher function